server/apps/prepopulate/data_initialization/validators.json

```diff
@@ -41,10 +41,16 @@
         },
         "subject" : {
             "type" : "list",
-            "required" : true
+            "required" : true,
+            "minlength" : 1
```

Comment:
I think it is still needed. With required empty strings are validated. 
https://cerberus.readthedocs.org/en/latest/#required